vue-app/src/views/VerifyLanding.vue

```diff
@@ -6,52 +6,54 @@
-            We use BrightID to stop bots and cheaters, and make our funding more
-            democratic.
-          </div>
-          <h2>What you'll need</h2>
-          <ul>
-            <li>
-              BrightID – available on
-              <a
-                href="https://apps.apple.com/us/app/brightid/id1428946820"
-                target="_blank"
-              >
-                iOS</a
-              >
-              or
-              <a
-                href="https://play.google.com/store/apps/details?id=org.brightid"
-                target="_blank"
-                >Android</a
-              >
-            </li>
-            <li>An Ethereum wallet</li>
-            <li>Access to Zoom or Google Meet</li>
-          </ul>
-          <router-link to="/sybil-resistance/"
-            >Why is this important?</router-link
+      </div>
+    </div>
+    <div class="content">
+      <div class="flex-title">
+        <h1>Prove you’re only using one account</h1>
+      </div>
+      <div class="subtitle">
+        We use BrightID to stop bots and cheaters, and make our funding more
+        democratic.
+      </div>
+      <h2>
+        What you'll need
+        <tooltip
+          position="top"
+          content="If you've previously donated to a CLR round, use the same wallet to bypass some BrightID steps"
+          ><img width="16px" src="@/assets/info.svg" class="info-icon"
+        /></tooltip>
+      </h2>
+      <ul>
+        <li>
+          BrightID – available on
+          <a
+            href="https://apps.apple.com/us/app/brightid/id1428946820"
+            target="_blank"
           >
-          <div v-if="isRoundFullOrOver" class="warning-message">
-            The current round is no longer accepting new contributions. You can
-            still get BrightID verified to prepare for next time.
-          </div>
-          <div class="btn-container">
-            <wallet-widget v-if="!currentUser" :isActionButton="true" />
-            <router-link
-              v-if="currentUser"
-              to="/verify/connect"
-              class="btn-primary"
-              >Get started</router-link
-            >
-            <router-link to="/" class="btn-secondary">Go home</router-link>
-          </div>
-        </div>
+            iOS</a
+          >
+          or
+          <a
+            href="https://play.google.com/store/apps/details?id=org.brightid"
+            target="_blank"
+            >Android</a
+          >
+        </li>
+        <li>An Ethereum wallet, with enough gas for two transactions</li>
+        <li>Access to Zoom or Google Meet</li>
+      </ul>
+      <router-link to="/sybil-resistance/">Why is this important?</router-link>
+      <div v-if="isRoundFullOrOver" class="warning-message">
+        The current round is no longer accepting new contributions. You can
+        still get BrightID verified to prepare for next time.
+      </div>
+      <div class="btn-container">
```

Comment:
I think that if `isRoundFullOrOver` we shouldn't display this div right? because we are not accepting new contributors.

Reply:
Or maybe we allow him to enter the verification process but he will not be able to register in the last step. I guess this is more appropriate

Reply:
Right, we warn them but still allow them to get verified.

Reply:
Does fill a _bit_ weird though, as the user might not understand why the last step/tx will fail 🤔 .